Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -417,6 +417,9 @@ - (void)_chooseMediaItems:(NSArray<_WKFileUploadItem *> *)mediaItems
 
     NSUInteger imageCount = mediaItems.count - videoCount;
 
+    // FIXME: Match the macOS behavior of showing the file name in the case of a single file once.
+    // Currently, the file names are UUIDs which are not pleasant to display, so this should be
+    // done once the files are saved using their actual names, which will be done once 245906 is resolved
```

Comment:
Should we file a separate bug for this? It's unclear whether this will be addressed in 245906, or in a separate change after 245906 has been fixed.

(also missing a period at the end of the sentence)